.env.ci

```diff
@@ -0,0 +1,41 @@
+JEST_TIMEOUT=90000
```

Comment:
Can we add a comment to the top of this file without breaking it?
Would be good to describe when its used how it needs to be kept up to date etc.